go.mod

```diff
@@ -11,7 +11,6 @@ require (
 	github.com/mongodb/grip v0.0.0-20220401165023-6a1d9bb90c21
 	github.com/pkg/errors v0.9.1
 	github.com/stretchr/testify v1.8.1
-	go.mongodb.org/mongo-driver v1.10.3
+	go.mongodb.org/mongo-driver v1.10.3 // indirect
```

Comment:
I believe because it is a dependency of a direct dependency here, but does not have a use go modules:

```
go.mod file only records the direct dependency. However, it may record an indirect dependency in the below cases

Any indirect dependency which is not listed in the go.mod file of your direct dependency or if direct dependency doesn’t have a go.mod file, then that dependency will be added to the go.mod file with //indirect as the suffix
Any dependency which is not imported in any of the source files of the module.
```

https://golangbyexample.com/direct-indirect-dependency-module-go/

Reply:
I wonder which dependency imports it, though.
Like if I remove the mongo-driver dependency from go.mod and run `go mod tidy` it seems like everything's fine.

Reply:
You are right! Not sure why mod tidy wouldn't clean those up automatically.